benchmarks/repord.md

```diff
@@ -0,0 +1,38 @@
+
+TorchServe Benchmark on gpu
```

Comment:
I would list the full .mar and input so the data is easier to copy to someplace like excel for further analysis